Chirp/OTP Components/PasscodeBoxView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  PasscodeBoxView.swift
+//  Chirp
+//
+//  Created by Stepan Kukharskyi on 6/5/23.
+//
+
+import UIKit
+
+class PasscodeBoxView: UIView {
+    
+    private let valueLabel = UILabel()
+    
+    var value: String? {
+        didSet { valueLabel.text = value ?? "" }
+    }
+    
+    var activeColor: CGColor = #colorLiteral(red: 0.337254902, green: 0.1921568627, blue: 0.1450980392, alpha: 1)
+    var inActiveColor: CGColor = #colorLiteral(red: 0.8509803922, green: 0.8509803922, blue: 0.8509803922, alpha: 1)
```

Comment:
Great thinking to keep these here! 💡
When @uzoh merges his PR we will update these to use the UIColor objects he has made so that everything points to one UIColor object.

Reply:
noted